processing/src/main/java/org/apache/carbondata/processing/util/CarbonDataProcessorUtil.java

```diff
@@ -375,6 +375,25 @@ public static boolean isHeaderValid(String tableName, String[] csvHeader,
     return type.toArray(new DataType[type.size()]);
   }
 
+  /**
+   * get visible no dictionary dimensions as per data field order
```

Comment:
i think better to avoid mentioning dictionary

Reply:
all these functions are replicated from the same file. Whole sort step has dictionary and no-dictionary mapping. 
I do agree that we don't have dictionary. But cannot change all the current sort step variables and logic in this PR.